cnd/src/swap_protocols/swap_id.rs

```diff
@@ -35,3 +36,10 @@ impl fmt::Display for SwapId {
         self.0.fmt(f)
     }
 }
+
+impl Scribe for SwapId {
+    fn scribe(&self) -> String {
+        let s = format!("{}", self);
+        s[..8].to_string() // Arbitrary number of human readable digits.
```

Comment:
:mega: We should always display the full swap id. Being a UUID, it is already human readable.

Reply:
Surely we do not need to log 128 bits just to show a unique swap - its way too verbose.

Reply:
But it removes the ability to take the swap-id from the outside and do a search in the logs.

Reply:
What do you mean, if we use the first 12 characters of the swap_id then one can still grep for them.  This is inline with how we use `--abbrev` in the kernel to pass around commit hashes as only 12 characters in email.

Reply:
To me, it seems like this would be equivalent of `git` would only emit 12 char commit hashes in every log and if someone passes you a full commit hash, you need to first abbreviate it if you want to search `git log` for it.

I am not a big fan of this. It is something you have to teach someone once they are surprised that they cannot just take a swap id and grep a log for it.

Reply:
Ok, I have removed `Scribe` and implemented `Display` on things that we want to log.  Of note the implementation for `Request` is super minimal, adding the ledgers and assets is going to be a punish because of all the traitbounds `<AL, BL, AA, BA>`.  This _should_ be resolved if/when we move to tracing since these can be a part of the context.